File: OLSource1/v1/VS_officedev/codesnippet/CSharp/extending-word-documents-and-excel-workbooks-in-vsto-add-ins-at-run-time_3.cs
            Microsoft.Office.Interop.Excel.Worksheet nativeWorksheet =
                Globals.ThisAddIn.Application.ActiveSheet;
            if (nativeWorksheet != null)
            {
                Microsoft.Office.Tools.Excel.Worksheet vstoSheet = 
                    Globals.Factory.GetVstoObject(nativeWorksheet);
            }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/extending-word-documents-and-excel-workbooks-in-vsto-add-ins-at-run-time_3.vb
        Dim NativeWorksheet As Microsoft.Office.Interop.Excel.Worksheet =
            Globals.ThisAddIn.Application.ActiveSheet
        If NativeWorksheet IsNot Nothing Then
            Dim vstoSheet As Microsoft.Office.Tools.Excel.Worksheet =
                Globals.Factory.GetVstoObject(NativeWorksheet)
        End If